### DISCUSSION TITLE
Script needs to be above x-data?

### DISCUSSION THREAD

**Discussion by FredMar** [alpinejs/alpine#3446] [1. Help] [answered]
This only works if the script tag is above the x-data:

![image](https://user-images.githubusercontent.com/16269671/222951946-7589438f-3064-4099-a5fc-fa84eaa3ca5a.png)

Is there anything to do about that?

Thanks :)

**Comment by SimoTod** [comment, marked as answer]
It also works if the script tag goes after.

Chances are that you didn't defer the script tag including alpine.

https://alpinejs.dev/start-here

**Reply by FredMar** [reply]
And of course you are right - thanks a lot for your help! 👍
